### ISSUE TITLE
Feature Request

### ISSUE THREAD

**Issue by tberneman** [mzomparelli/zVirtualDesktop#237]
When you right click the system tray icon and choose "Switch Desktop" I would like it to display the Title of the desktop as well. ex: "Desktop 8 - Social & Calendar   CTRL+WIN+NumPad8"

**Comment by mzomparelli** [comment]
Good idea. Seems like it could be a lot of text for a menu item so I need to think of a better way. Maybe a custom menu item that is taller so I can fit sub text or maybe I remove the number from the name and use the icons instead.

**Comment by tberneman** [comment]
Maybe use "^ + WIN + 4" instead of "CTRL + WIN + WinPad4"?

Alternatively you could use "^⊞2" or "^ ⊞ 2" or "^ + ⊞ + 2" and at the
bottom of the popup menu below "New Window" you could put "⊞ is the Windows
key." or something like that.

Here's a link for that character:
http://www.fileformat.info/info/unicode/char/229e/index.htm

And Wikipedia uses it describing windows shortcuts here:
https://en.wikipedia.org/wiki/Windows_key

Tim :o]

**Comment by mzomparelli** [comment]
Please check out version 1.0.95.4. I've added the desktop name. Testing it out to see how it goes.

**Comment by tberneman** [comment]
Hi Michael,

That feature has been in for a long time now. The feature I'm ask for is to
show the name here:
[image: image.png]
My first 2 desktops are named "Main" and "Gaming". I want to see that in
"Switch desktop" submenu where it says "Desktop 1" and "Desktop 2".

Tim :o]

**Comment by mzomparelli** [comment]
Check that you have 1.0.95.4. It should be as you describe.

**Comment by tberneman** [comment]
My bad. I tried the "Check for Updates" and I should have gone to GitHub
and downloaded it from there. I did that this morning and it looks good! If
we could shorten the keyboard shortcut or widen the box to show more
characters that would be great. If not you can still get the gist of the
desktop names and it works great! Good Job! :)

**Comment by mzomparelli** [comment]
I plan on tweaking but this is a start.
